src/app/services/score.service.ts

```diff
@@ -11,13 +10,13 @@ export class ScoreService {
 
   constructor(private mqttService: MqttService) { }
 
-  score(): Observable<Score> {
-    return this.mqttService.observe('game/score').pipe(switchMap(message =>  {
-      return of(JSON.parse(message.payload.toString()));
+  score(teamId: number): Observable<number> {
+    return this.mqttService.observe('game/score/' + teamId).pipe(switchMap(message =>  {
```

Comment:
please replace game/score by team/score (was wrong described in the message overview)